packages/@apphosting/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apphosting/common",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
You'll probably want to change both adapters to use this new version too

Reply:
will do in separate PRs!